### DISCUSSION TITLE
[Feature] "Last uploaded" album folder

### DISCUSSION THREAD

**Discussion by temal32** [immich-app/immich#14563] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello, Immich sorts the photos/videos by newest date. But what about a new album named "Last uploaded" just like the album "Last saved" in the iPhone photos app. This album doesn't sort photos by data but by last uploaded to the Immich server.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#8716

**Reply by temal32** [reply]
oh damn, sorry
